docs/source/schema/complex_types/record.rst

```diff
@@ -2,11 +2,10 @@
  record
 ########
 
-The ``record`` type represents a labeled array capable of holding any data type, just like the
-struct in C++ or the Series in pandas.
+The ``record`` type is where we can definte complext data stuctures by grouping related variables togather in the same place. It is simiilar to 
```

Comment:
`complext` -> `complex`
`togather` -> `together`
`simiilar` -> `similar`

Reply:
Portex doc use the third person. shall we remove the `we` in the doc?

Reply:
`definte` -> `define`

Reply:
`stuctures` -> `structures`

Reply:
Done